geopandas/geodataframe.py

```diff
@@ -1676,41 +1676,28 @@ def explode(self, column=None, ignore_index=False, index_parts=None, **kwargs):
 
         df_copy = self.copy()
```

Comment:
Unrelated to the other changes, but I think this initial copy should not be needed actually (since we first do a drop and take, before using it to create the result, and those operations should always result in a copy as well)

Reply:
I've removed it.